metaflow/plugins/argo/argo_live_run.py

```diff
@@ -62,7 +62,7 @@ def trigger(
         flow_name: str = None,
         parameters: dict = None,
         wait: bool = True,
-        wait_timeout: int = 30,  # in minutes TODO: determine proper timeout
+        wait_timeout: int = 60 * 12,  # in minutes TODO: determine proper timeout
```

Comment:
Any reason why 12 hour of time out is needed? I am not sure what default timeout should be used. I'm most trying to understand your thought process.

Reply:
It was just a guess as to what's best. I've heard of some flows taking multiple hours to run, so I wanted to be conservative and not cut off any flows before they are finished. I kept in a TODO about this because I don't know what the best default setting is.

Reply:
I added a note about this in our 1:1 backlog

Reply:
I think 12 hour is good for not having user worrying about setting a different value. The downside is, if the triggered flow stuck in runner state for any reason and customer didn't expect run time to be that long, this function will wait for a long time

For cases where user needs a shorter timeout, they have better idea of how long a wait is needed.

Reply:
Sounds good. I'll leave it at 12 hours for the default value